lectures/olg.md

```diff
@@ -735,15 +732,15 @@ def k_update(k, model):
     return optimize.newton(lambda k_prime: f(k_prime, k, model), 0.1)
 ```
 
-Finally, here is the 45 degree diagram.
+Finally, here is the 45-degree diagram.
 
 ```{code-cell} ipython3
 kmin, kmax = 0, 0.5
-x = 1000
-k_grid = np.linspace(kmin, kmax, x)
+n = 1000
+k_grid = np.linspace(kmin, kmax, n)
 k_grid_next = np.empty_like(k_grid)
 
-for i in range(x):
+for i in range(n):
```

Comment:
see above comment

Reply:
Hi @mmcky, in the exercise, we define `k_update` again as follows,

```Python
def k_update(k, model):
    return optimize.newton(lambda k_prime: f(k_prime, k, model), 0.1)
```
 but this time, the `k_update` function is not inherently vectorized as in the main text, which is
```Python
def k_update(k, α, β):
    return β * (1 - α) * k**α /  (1 + β)
```
So we cannot use the way in the main text to vectorize `k_grid_next`.

Best,
Longye